test/cypress/integration/03-expenses.page.test.js

```diff
@@ -1,3 +1,106 @@
+import { randomSlug } from '../support/faker';
+
+describe('New expense flow ', () => {
+  const collectiveSlug = randomSlug();
+
+  function SubmitExpense(description, tag) {
```

Comment:
Let's keep function names lowercase

```suggestion
  function submitExpense(description, tag) {
```